com.mastermindzh.tidal-hifi.metainfo.xml

```diff
@@ -116,7 +130,7 @@
     <release version="5.0.0" date="2023-04-22">
       <description>
         <p>Adblocking support thanks to @thanasistrisp.</p>
-        <p> Replaced "muting artists" with a full implementation of an Adblock mechanism<br /> >
+        <p> Replaced "muting artists" with a full implementation of an Adblock mechanism<br />
```

Comment:
metainfo check is very picky and doesn't like br tags

Reply:
Yeah, figured...
Weird thing is that it was already in the code: https://github.com/flathub/com.mastermindzh.tidal-hifi/blob/master/com.mastermindzh.tidal-hifi.metainfo.xml#L119

Reply:
Might have been due to recent changes :shrug:

Reply:
we'll see, I removed it... let's hope it builds :)